File: PPA/DailyFlash/July/29July/Program10.c


#include<stdio.h>

void main() {
	int num=1;
	for(int i=1; i<=5; i++) {
		
		for(int j=1; j<i; j++) {

			printf("\t");
		}

		for(int k=5; k>=i; k--) {

			if(i==1 || i==3 || i==5)
				printf("%d\t",num * num);
			else
				printf("%d\t",num*num*num);
			num++;
		}
		printf("\n");
	}
}

File: Java/Assignments/Assignment/CombineAssignment/Program10.java

/*	Program 10: Write a java program to calculate the power of a given number.
	
	Input: number 10, power: 2
	Output: 10 to the power 2 is 100
*/

class PowerOfNumber {

	public static void main(String[] args) {

		int num = 10;
		int power = 2;
		int ans=1;

		while(power > 0) {

			ans = ans * num;
			power--;
		}
		System.out.println("10 to the power 2 is "+ans);
	}
}
